queue_job/models/queue_job.py

```diff
@@ -267,8 +267,8 @@ class JobChannel(models.Model):
     @api.depends('name', 'parent_id.complete_name')
     def _compute_complete_name(self):
         for record in self:
-            # if not record.name:
-            #     return  # new record
+            if not record.name:
+                return  # new record
```

Comment:
Should it be `continue`? Or better to use `self.filtered('name')` in the loop

Reply:
Good catch :) (however continue is better than `self.filtered` as anyway have to loop on the records, `filtered` would use 2 loops)